### DISCUSSION TITLE
Give dev option of using PowerPlatform.Dataflows or PowerBI.Dataflows when importing a table, and workspace name in the UI

### DISCUSSION THREAD

**Discussion by edhans** [TabularEditor/TabularEditor3#1033] [Ideas]
When importing a table from a Power BI dataflow, TE/3.6.0 currently has two issues:

1. You cannot see what workspace you are in if you pick the dataflow connector from an existing source. In this model I have dataflows from two different workspaces.

Which workspace is this connecting to?
![image](https://user-images.githubusercontent.com/45298358/236257282-b22f01d3-7133-4ea9-9c43-b16192f7c918.png)

Once in the workspace, which one am I in?
![image](https://user-images.githubusercontent.com/45298358/236257440-49e1cf42-b540-4b3a-8f6c-0078b78c619d.png)

2. Once you pick a table, we should be given the option of using the PowerPlatform.Dataflow() connector as that one is required for incremental refresh or any filtering that would be folded back to the Dataflow server.

![image](https://user-images.githubusercontent.com/45298358/236257995-066633a0-b119-4132-8837-e752b253b57e.png)

Note: Once you've imported it you can tell as the M code TE generates has the WS name, which I love. Doing this in Power BI you only get GUIDs. Such a pain, so the comment text to the right is fantastic!

**Comment by DBojsen** [comment]
Hi @edhans 

Thank you for the suggestion.  We're evaluating it and will get back to you.

BR David - TE3 Support

**Comment by DBojsen** [comment]
Hi again @edhans 

We have discussed internally, and have added an item to our backlog to improve the DataFlows experience.
We have not completed the design of it yet, so I cannot comment yet what will be included, but will get back to you.

BR David - TE3 Support
